include/gz/math/MassMatrix3.hh

```diff
@@ -930,6 +930,82 @@ namespace gz
         return this->SetMoi(R * L * R.Transposed());
       }
 
+      /// \brief Set inertial properties based on a Material and equivalent
+      /// cone aligned with Z axis.
+      /// \param[in] _mat Material that specifies a density. Uniform density
+      /// is used.
+      /// \param[in] _length Length of cone along Z axis.
+      /// \param[in] _radius Radius of cone.
+      /// \param[in] _rot Rotational offset of equivalent cone.
+      /// \return True if inertial properties were set successfully.
+      public: bool SetFromConeZ(const Material &_mat,
+                                    const T _length,
+                                    const T _radius,
```

Comment:
indentation

Reply:
I copied this indentation style from SetFromCylinderZ to keep it under 80 char len:
https://github.com/gazebosim/gz-math/blob/118af0968fcb131338f9ebb9bba881ce43dbaa96/include/gz/math/MassMatrix3.hh#L1017-L1044